processing_handler.go

```diff
@@ -60,8 +61,12 @@ func setCacheControl(rw http.ResponseWriter, originHeaders map[string]string) {
 	}
 
 	if len(cacheControl) == 0 && len(expires) == 0 {
-		cacheControl = fmt.Sprintf("max-age=%d, public", config.TTL)
-		expires = time.Now().Add(time.Second * time.Duration(config.TTL)).Format(http.TimeFormat)
+		ttl = config.TTL
+		if _, ok := originHeaders["Fallback-Image"]; ok {
```

Comment:
Ugh, I forgot we need to check `config.FallbackImageTTL` here.

```suggestion
		if _, ok := originHeaders["Fallback-Image"]; ok &&  config.FallbackImageTTL > 0 {
```